source/imaer-gml/src/main/java/nl/overheid/aerius/gml/base/characteristics/OPSSpreadUtil.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright the State of the Netherlands
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see http://www.gnu.org/licenses/.
+ */
+package nl.overheid.aerius.gml.base.characteristics;
+
+import nl.overheid.aerius.shared.domain.v2.geojson.Geometry;
+import nl.overheid.aerius.shared.domain.v2.geojson.LineString;
+import nl.overheid.aerius.shared.domain.v2.geojson.Point;
+
+/**
+ * Utility to handle the legacy variant of handling the OPS spread value.
+ *
+ * In older versions of OPS the spread parameter in the input data for point sources was ignored and calculated within OPS as being half the
+ * emission height.
+ * With newer OPS versions (since 5.3.1) the spread value in the OPS input file is read and not calculated.
+ * This means the application should pass the desired spread value.
+ * For older IMAER versions (prior to 6) for point sources it means that value is derived from the emission height.
+ * Even if the spread value of a point source is set in the IMAER file it will be ignored.
+ * For none point sources and if spread was not set in older IMAER versions (< 6.0)the sector default will be used.
```

Comment:
```suggestion
 * For non-point sources, and if spread was not set in older IMAER versions (< 6.0), the sector default will be used.
```
(Not too sure about the commas, I just think it makes it makes sense this way...)